packages/common/src/lib/common/tools.ts

```diff
@@ -2682,7 +2683,7 @@ export function formatAliasValue(
             sourceObj.max !== undefined
         ) {
             // scale target between 0 and 100 % based on sources min/max
-            state.val = ((state.val - sourceObj.min) / (sourceObj.max - sourceObj.min)) * 100;
+            state.val = (((state.val as any) - sourceObj.min) / (sourceObj.max - sourceObj.min)) * 100;
```

Comment:
We should think about making `ioBroker.State` generic, so `state.val` has the correct type. Not in this PR though.

Reply:
good idea :+1: